terminus-ui/csv-entry/src/csv-entry.component.ts

```diff
@@ -1,3 +1,7 @@
+// NOTE: Moving items like `getHeaderCellName` into the template would add too much logic to the template
+// tslint:disable: template-no-call-expression
```

Comment:
Currently there is no way to disable a tslint rule for part of an HTML file. So that rule is either all on or all off. In this case, refactoring to remove any function calls would be too large a change to do in this ticket. And frankly I'm a little unsure the value that would bring. So for now we just disable the rule completely for the entire component.